cloudsql/sqlserver/database-sql/README.md

```diff
@@ -0,0 +1,120 @@
+# Connecting to Cloud SQL (SQL Server) from a Go web app
```

Comment:
We generally try to avoid having in depth READMEs in this repository. Instead, we prefer having the canonical documentation on cloud.google.com. We've had issues in the past with READMEs falling out of date and not working. So, we've been switching to the README only having a link to the corresponding cloud.google.com tutorial.

Reply:
This REAME is the canonical instructions. These samples deviate a bit from the traditional approach - the snippets being shown on c.g.c (in multiple places) are a small part of the overall app, but provide a link to the README for their context in a complete web application. 

We previously [discussed it for the MySQL sample as well](https://github.com/GoogleCloudPlatform/golang-samples/pull/1085#discussion_r347005538).